src/test/checksum/file1

```diff
@@ -1,2 +1,3 @@
 Hello.  This is a simple text file with a known checksum.
 The checksum unit test is run on this file to check for regressions.
+
```

Comment:
This file is passed to checksumming functions which expect input size to be a multiple 4. Its size is 127 bytes, so there's mismatch.